cypress/tests/ImageElement.spec.ts

```diff
@@ -224,13 +238,27 @@ describe("ImageElement", () => {
         setDocFromHtml(values);
         assertDocHtml(getSerialisedHtml(values));
       });
+
+      it.only("should select all of its text when the select all shortcut is used", () => {
+        addImageElement({
+          caption: "The remaining field.",
+          src: "Here is some text we'd like to delete.",
+        });
+        typeIntoElementField("src", "{cmd+a}{backspace}");
+
+        getElementRichTextField("src").should("have.text", "");
+        getElementRichTextField("caption").should(
+          "have.text",
+          "The remaining field."
+        );
+      });
     });
 
     describe("Checkbox field", () => {
       it(`should be clickable`, () => {
-        addImageElement();
-        getElementField("useSrc").find("input").click();
-        getElementField("useSrc").find("input").should("be.checked");
+        addImageElement({});
+        getElementRichTextField("useSrc").find("input").click();
+        getElementRichTextField("useSrc").find("input").should("be.checked");
```

Comment:
Is this change related?

Reply:
No - not sure how I managed that. Will fix.